.travis.yml

```diff
@@ -11,4 +12,4 @@ install:
   - "pip install -r requirements.txt"
   - "pip install ."
 script:
-  - py.test
+  - tox
```

Comment:
This makes travis run the tests for Python 2.7 and Python 3.5 for each job (every combination of Python and requests versions listed above) in the build matrix. 

- [ ] Remove extra Python versions from `travis.yml`. Leave only 3.5.
- [ ] Remove testing against different requests versions. Those versions listed here are really outdated (current version is 2.9.1) and testing against each minor version makes the build take unnecessarily long.
- [ ] Add `TESTENV` environment variable for each Python version we want to test against (2.7, 3.5, pypy) and change the test command to `tox -e $TESTENV` (see [pytest's travis.yml](https://github.com/pytest-dev/pytest/blob/master/.travis.yml) for an example).